docs/api/browser-window.md

```diff
@@ -1167,3 +1167,39 @@ Clears the session's HTTP cache.
 * `callback` Function - Called when operation is done
 
 Clears the data of web storages.
+
+### Session.setProxy(config, callback)
+
+* `config` String
+* `callback` Function - Called when operation is done
+
+Parses the `proxy-rules` from a string, indicating which proxies to use for the session.
```

Comment:
We should use `config` instead of `proxy-rules` in the docs, since `config` is the parameter's name.